config/read.go

```diff
@@ -23,6 +23,14 @@ import (
 
 const DefaultAPIBaseURL = "https://api.pganalyze.com"
 
+func parseConfigBool(value string) bool {
+	if value == "" || value == "0" || value == "off" || value == "false" || value == "no" || value == "f" || value == "n" {
```

Comment:
Worth doing this comparison against the lower-cased version of string? (both go-ini and Postgres seem to handle this case-insensitively)

Reply:
Yes! I had that change locally (the commit message even mentions it), but neglected to push it. Updated.